tools/revert-release.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# CAUTION!
+# This script does one thing, which is to revert stable tag in WordPress.org svn to the prior tag.
+# It should only be used in extreme emergency cases.
+
+# Check for requirements. Using `jq` to easily parse json. Is there a simpler way to get the latest tags?
+if [ -z $(command -v jq) ]; then
+	echo "This script requires the jq library."
+	read -p "Do you want to install it on your system with Homebrew? [y/N]" -n 1 -r
+	if [[ $REPLY != "y" && $REPLY != "Y" ]]; then
+		exit 1;
+	else
+		brew install jq 2>/dev/null
+		echo "Done!"
+	fi
+fi
+
+YELLOW='\033[1;33m'
+RED='\033[0;31m'
+NOCOLOR='\033[0m'
+
+# Current stable version
+CURRENT_STABLE_VERSION=$(curl -s http://api.wordpress.org/plugins/info/1.0/jetpack.json | jq -r .version)
+
+# Get all versions, strip anything with alpha characters such as -beta or trunk.
+LAST_STABLE_TAG=$(curl -s http://api.wordpress.org/plugins/info/1.0/jetpack.json | jq -r '.versions | delpaths([paths | select(.[] | test("[A-Za-z]+"; "i"))]) | keys[-2]' )
```

Comment:
Is there a better/simpler way to get the most recent tag this that doesn't use a third party lib (`jq`)?

Reply:
`jq` seems to be the easiest way to parse json from what I can tell. I'd be okay requiring it in these cases.

Reply:
Yeah it's the best I found too. It's just horribly un-readable code.